### PR TITLE
Fix Android Emulator Hangs

### PR DESCRIPTION
**Pull request by kalder**
In some cases, Android Emulators will hang because of output buffering. Closing the pipe fixes this.

There's also some unused code (stderr pipe, log tempfile) that should be removed.

### CODE REVIEW

**Review by oliverchang — APPROVED** [review]
nice! do we need to prevent stderr from filling up too?

File: src/python/platforms/android/emulator.py [hunk 1]
```diff
@@ -82,8 +77,7 @@ def run(self):
       raise EmulatorError('Attempted to `run` emulator before calling `create`')
 
     logs.log('Starting emulator.')
-    self.process = self.process_runner.run(
-        stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    self.process = self.process_runner.run(stdout=subprocess.PIPE)
```
Comment (oliverchang) [inline src/python/platforms/android/emulator.py:80]: Since we don't care about stderr, let's set stderr to open(os.devnull, 'w').

Comment (kalder) [reply src/python/platforms/android/emulator.py]: Done, but I'm using subprocess.DEVNULL.

**Pull request merged**